### PR TITLE
[readme]: Revert #7795, disable some rules in linter

### PR DESCRIPTION
**Pull request by supechicken**
Revert #7795 #7796 

We should not follow the linter unless an alternative or better way exists... Instead, modify rules to fit our needs.

### Changes
- Disable pure markdown rule, we need HTML to do styling like centering things
- Move light/dark mode logic to a magic `.svg` file to keep readme clean as @uberhacker suggested
- Remove instructions about installing on google onhub (as we cannot make sure if that still works)
- Align images to make them look better

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,14 +91,20 @@ Available packages are listed in the [packages directory](https://github.com/chr
 
 Chromebrew will wipe its `BREW_DIR` (`/usr/local/tmp/crew` by default) after installation unless you pass `-k` or `--keep` when running `crew install`.
 
-```bash
+```text
```
Comment (uberhacker) [inline README.md:94]: The `bash` passed with the previous rules?

**Review by uberhacker — CHANGES_REQUESTED** [review]
See comments.

File: README.md [hunk 2]
```diff
@@ -53,18 +53,6 @@ Then download and run the installation script below:
...
-```bash
-# passwd chronos
-Changing password for chronos.
-Enter new UNIX password:
-Retype new UNIX password:
-# su - chronos
-Password:
-$ curl -Ls git.io/vddgY | bash
-```
-
```
[the remark this answers is not in the document]
Comment (supechicken) [reply README.md:67]: As we cannot make sure if that still works

File: .mdl_style.rb [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
-exclude_rule 'MD013'
-# Ignore horizontal rule style.
-exclude_rule 'MD035'
-# Ignore emphasis used instead of a header.
-exclude_rule 'MD036'
-# Ignore first line in file should be a top level header.
-exclude_rule 'MD041'
-# Ignore code block style.
-exclude_rule 'MD046'
+rule 'MD003', style: :atx
```
[the remark this answers is not in the document]
Comment (supechicken) [reply .mdl_style.rb:2]: Yes but this line appends the `style=atx` option (which overrides the default one)

File: README.md (hunk 1 above)
Comment (supechicken) [reply README.md:94]: Yes but the `<` and `[` symbol will break syntax highlighting

**Review by uberhacker — APPROVED** [review]

**Pull request merged**